pkg/protoc-gen-go/main.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
```

Comment:
We can't add this code, it's BSD licensed.

I think it's better just to commit the generated .pb.go file as in the other patch.